### ISSUE TITLE
Error with callTumorOnly

### ISSUE THREAD

**Issue by namsyvo** [tgen/LumosVar#3]
Hi,
I'm trying to run callTumorOnly, but I got the following error:

Starting parallel pool (parpool) using the 'local' profile ... 
connected to 16 workers.
message =
finished processing chromosomes
Undefined function or variable 'AllData'.
Error in preprocessTumorOnly (line 132)
Error in TumorOnlyWrapper (line 42)
MATLAB:UndefinedFunction

I've searched the variable "AllData" in the source code and found only one fine that includes "AllData", preprocessTumorOnly. However in this file, "AllData" is not defined before using. Is it a bug or I did something wrong?

Also, I used binary file to run (bin/callTumorOnly), is it compiled from this version of source code?
Thanks.

**Comment by rhalperin** [comment]
It looks like it isn't reading in any data.  You should also get a log file that will have the prefix of the "outName" parameter defined in the yaml and the extension "log.txt".  What do you see in the log file?

**Comment by namsyvo** [comment]
Oh, I'm sorry I posted the wrong message (the previous one was an error as well but I have fixed it, it's due to wrong data file/directory), the error I want to ask is the the following:

inputParam = 
     ....

Starting parallel pool (parpool) using the 'local' profile ... connected to 16 workers.
Error using preprocessTumorOnly (line 42)
Subscripted assignment dimension mismatch.
Error in TumorOnlyWrapper (line 42)
MATLAB:subsassigndimmismatch

You can look at the log in attached file. Thanks.
[tcga_fa_8693_10_log.txt](https://github.com/tgen/LumosVar/files/1746049/tcga_fa_8693_10_log.txt)

**Comment by rhalperin** [comment]
From the log file, it looks like at most it is finding a read depth of 1 in the region 10:46841-48136.  Do you see reads in that region when you look at it in IGV?

**Comment by namsyvo** [comment]
Yes, I still see reads at that region. You can find it in attached file
![igv_snapshot](https://user-images.githubusercontent.com/1686614/36513397-09013bc0-1735-11e8-9811-f56ebda3a013.png)

**Comment by rhalperin** [comment]
Hmm, that is strange that it is seeing a read depth of 0-1 in that region.  Could you please make a mini bam of that region and send it to me and I will take a look?

**Comment by namsyvo** [comment]
If I understand correctly, you mean it found "at least" a read depth of 0-1 in that region? If so it could, because there are no reads in 10:46841-46855 (a small region at left-most of the above image). Does it matter?
Could you give me your email so that I can send the file through email? Due to some reason, I don't want to send the file publicly here. Thanks.

**Comment by rhalperin** [comment]
Sure you can email me at rhalperin at tgen dot org.  I suspect there may be a bug in how it is handling a region without any reads.

**Comment by namsyvo** [comment]
Hi,
Have you looked at the errors I sent you? Thanks.
Nam

**Comment by rhalperin** [comment]
Sorry, I have not gotten to it yet.  I've had a busy week preparing for a conference.  I should have time next week.  I apologize for the delay.
